service/controller/v12/adapter/launch_configuration_test.go

```diff
@@ -94,7 +94,7 @@ func TestAdapterLaunchConfigurationSmallCloudConfig(t *testing.T) {
 		},
 		{
 			description:  "s3 http uri",
-			expectedLine: `s3_http_uri="https://s3.myregion.amazonaws.com/000000000000-g8s-test-cluster/cloudconfig/v_3_3_1/$USERDATA_FILE"`,
+			expectedLine: `s3_http_uri="https://s3.myregion.amazonaws.com/000000000000-g8s-test-cluster/cloudconfig/v_3_3_2/$USERDATA_FILE"`,
```

Comment:
Is this test actually useful? I am thinking to remove this.

Reply:
I removed this test case but see below I refactored the s3object test to use a regex. So I don't think we lose much by removing this case.